packages/xsnap/src/xsnap.js

```diff
@@ -194,10 +217,11 @@ export function xsnap(options) {
 
   /**
    * @param {string} message
-   * @returns {Promise<string>}
+   * @returns {Promise<{ reply: string, meterStats: any }>}
```

Comment:
again, any particular reason for `any` rather than `unknown`?

Reply:
Now typed.